README.md

```diff
@@ -10,9 +10,11 @@ It is mainly based on `AllenNLP` and `transformers`.
 ## Installation
 The following command installs all necessary packages:
 ```.bash
-pip install -r requirements.txt
+conda create --name <Environment_name> python=3.8
+conda activate <Environment_name> 
+python setup.py install
```

Comment:
I think we want `python setup.py develop`. But really I would probably say we should use `pip install -e .` or something along those lines. Also, this will install all dependencies using `pip`. Is that really what we want here?

For example, this is another way you might want to do it to utilize Conda.
```
conda install --name <<env_name>> python=3.8 --file requirements.txt
conda activate <<env_name>>
pip install -e . --no-dependencies
```

Reply:
Some of dependencies in the requirements are not supported by conda. i.e: `allenNLP 0.9.0` 
Would it be okay to just have `pip install -e .` after creating and activating env?

Reply:
I guess if it's not possible to satisfy all dependencies at this moment with Conda, there's no easy way. But perhaps just keep an eye on this issue.